src/DiskIO/IpcIo/IpcIoFile.cc

```diff
@@ -525,6 +533,19 @@ IpcIoFile::HandleNotification(const Ipc::TypedMsgHdr &msg)
         HandleResponses("after notification");
 }
 
+/// Handle messages in IPC queue without relying on the queue state being in
+/// sync with our state (because the queue may be reflecting the state of our
+/// killed predecessor). See also: CollapsedForwarding::HandleNewDataAtStart().
```

Comment:
```suggestion
/// killed predecessor).
/// \sa CollapsedForwarding::HandleNewDataAtStart()
```

Reply:
Done in 68b76da213296061e22b5a343b29466c8fa4ab3d.